packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -423,3 +423,17 @@ stage.grantManageConnections(lambda);
 // for all the stages permission
 webSocketApi.grantManageConnections(lambda);
 ```
+
+### API Keys
+
+Websocket APIs also support usage of API Keys. An API Key is a key that is used to grant access to an API. These are useful for controlling and tracking access to an API, when used together with [usage plans](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-api-usage-plans.html). These together allow you to configure controls around API access such as quotas and throttling, along with per-API Key metrics on usage.
```

Comment:
> An API Key is a key that is used to grant access to an API. These are useful for controlling and tracking access to an API, when used together with usage plans

How does one grant access based on an API based on the key and how can it be used together with a usage plan?

Reply:
I was thinking about this more, and I'm going to write some more examples in the readme of how to set these up.